third_party/txt/tests/paragraph_unittests.cc

```diff
@@ -432,6 +438,13 @@ TEST_F(ParagraphTest, DISABLE_ON_WINDOWS(RightAlignParagraph)) {
           paragraph->breaker_.getWidths()[paragraph->records_[0].line()],
       2.0);
 
+  // width_ takes the full available space, while tight_width_ wraps the glyphs
+  // as tightly as possible. Even though this text is more than one line long,
+  // no line perfectly spans the width of the full line, so tight_width_ is less
+  // than width_.
+  ASSERT_DOUBLE_EQ(paragraph->width_, available_width);
+  ASSERT_TRUE(paragraph->tight_width_ < paragraph->width_);
```

Comment:
You should be able to compare against either hardcoded values or the right edge of the right-most getRectsForRange box.

Reply:
Good call.